client/src/components/main/App/DemoDropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import { FormControl, Select, MenuItem, makeStyles } from "@material-ui/core";
+import React from "react";
+import {
+  assayedDemoList,
+  categoricalDemoList,
+  getDemoObject,
+} from "../../../services/main";
+import {
+  ClientAssayedFusion,
+  ClientCategoricalFusion,
+} from "../../../services/ResponseModels";
+
+const useStyles = makeStyles(() => ({
+  demoMenu: {
+    width: "200px",
+    height: "40px",
+    alignItems: "center",
+    backgroundColor: "white",
+    marginTop: "5px",
+    marginRight: "5px",
+  },
+}));
+
+interface DemoDropdownProps {
+  handleClear: () => void;
+  handleDemo: (fusion: ClientAssayedFusion | ClientCategoricalFusion) => void;
+}
+
+export default function DemoDropdown(
+  props: DemoDropdownProps
+): React.ReactElement {
+  const classes = useStyles();
+  const { handleClear, handleDemo } = props;
+  const [selectedDemo, setSelectedDemo] = React.useState("");
+  const demoIsSelected = selectedDemo && selectedDemo !== "none";
+
+  const path = window.location.pathname;
+  let demoData = assayedDemoList;
+  if (path.includes("/categorical-fusion")) {
+    demoData = categoricalDemoList;
+  }
+
+  const handleChange = (event) => {
+    const value = event.target.value;
+    setSelectedDemo(value as string);
+    if (value !== "none") {
+      getDemoObject(value).then((demoObject) => {
+        handleDemo(demoObject);
+      });
+    } else if (value === "none") {
+      handleClear();
+    }
+  };
+
+  return (
+    <FormControl variant="standard">
+      <Select
```

Comment:
IMO adding like 10px of padding to the left of the text in the selected box would be nice but I don't know how to do this in Select components properly (my attempts to get it right are all breaking in other ways)

![Screen Shot 2022-10-05 at 8 54 32 AM](https://user-images.githubusercontent.com/12942397/194065427-f884f40d-d720-4a20-a63b-a23e160da18f.png)

Reply:
done!